src/ice_shelf/MOM_ice_shelf_initialize.F90

```diff
@@ -395,14 +395,17 @@ end subroutine initialize_ice_shelf_boundary_channel
 !> Initialize ice shelf flow from file
 subroutine initialize_ice_flow_from_file(bed_elev,u_shelf, v_shelf,float_cond,&
                                          G, US, PF)
+!subroutine initialize_ice_flow_from_file(bed_elev,u_shelf, v_shelf,float_cond,ice_visc,&
+!                                         G, US, PF)
   type(ocean_grid_type), intent(in)    :: G    !< The ocean's grid structure
   real, dimension(SZDI_(G),SZDJ_(G)), &
                          intent(inout) :: bed_elev !< The bed elevation   [Z ~> m].
   real, dimension(SZIB_(G),SZJB_(G)), &
                          intent(inout) :: u_shelf !< The zonal ice shelf velocity  [L T-1 ~> m s-1].
   real, dimension(SZIB_(G),SZJB_(G)), &
                          intent(inout) :: v_shelf !< The meridional ice shelf velocity  [L T-1 ~> m s-1].
-
+!  real, dimension(SZDI_(G),SZDJ_(G)), &
+!                         intent(inout) :: ice_visc !< The ice vicscosity  [R_L4_Z_T~>kg_m2_s].
```

Comment:
More zombie comments, if unneeded I'd suggest removing them.

Reply:
If this line is not removed,  I would like to suggest revising the comment documenting the units of the ice viscosity to read something like `!< The ice viscosity  [R L4 Z T-1 ~> kg m2 s-1]` so that the units are unambiguous.

Reply:
Thank you, Bob. I'll correct these units in other places as well.